specification/app/resource-manager/Microsoft.App/stable/2022-05-01/SourceControls.json

```diff
@@ -182,7 +182,7 @@
               "$ref": "#/definitions/SourceControl"
             }
           },
-          "202": {
+          "201": {
```

Comment:
>"201" [](http://example.com/codeflow?start=10&length=5)

This is what the RP returns today, correct?

Reply:
because if it is not then this will be considered a breaking API change.

Reply:
Yes, now we implement it as 201.

Reply:
Has it always been 201 or we recently changed it in the RP?